### PR TITLE
update error info of ops，add some test cases for raise message

### PR DESCRIPTION
**Pull request by Steffy-zxf**
1. update error info of the ops (abs, acos, asin, atan, ceil, cos, exp, floor, log, pow, reciprocal, round, rsqrt, sin, sqrt, square, tanh)

2. add the unittests of the above refered ops (test error info)

### CODE REVIEW

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -8915,13 +8916,17 @@ def pow(x, factor=1.0, name=None):
             y_2 = fluid.layers.pow(x, factor=factor_tensor)
             # y_2 is x^{3.0}
     """
+    check_variable_and_dtype(x, 'x', ['float32', 'float64'], 'pow')
```
Comment (wawltor) [inline python/paddle/fluid/layers/nn.py]: pow op支持float32，float64, int32, int64类型

Comment (wawltor) [reply python/paddle/fluid/layers/nn.py]: 再check一下其它激活函数的op的数据类型支持

Comment (Steffy-zxf) [reply python/paddle/fluid/layers/nn.py]: 已修改

File: paddle/fluid/operators/activation_op.h [hunk 2]
```diff
@@ -68,9 +71,10 @@ inline void ExtractActivationTensor(const framework::ExecutionContext& context,
     *Out = context.Output<framework::Tensor>("Out");
   }
 
-  PADDLE_ENFORCE(*Out != nullptr,
-                 "Cannot get output tensor Out, variable name = %s",
-                 context.OutputName("Out"));
+  PADDLE_ENFORCE_NOT_NULL(
+      *Out, platform::errors::NotFound(
+                "Cannot get output tensor Out, variable name = %s",
```
Comment (chenwhql) [inline paddle/fluid/operators/activation_op.h]: cannot get tensor of output Variable Out?

File: paddle/fluid/operators/activation_op.h [hunk 3]
```diff
@@ -122,16 +130,18 @@ inline void ExtractActivationGradTensor(
...
+                              context.OutputName(framework::GradVarName("X"))));
 
   if (static_cast<int>(kDepValue) & static_cast<int>(kDepX)) {
     auto x_var = context.InputVar("X");
-    PADDLE_ENFORCE(x_var != nullptr,
-                   "Cannot get input tensor X, variable name = %s",
-                   context.InputName("X"));
+    PADDLE_ENFORCE_NOT_NULL(x_var,
+                            platform::errors::NotFound(
+                                "Cannot get input tensor X, variable name = %s",
```
Comment (chenwhql) [inline paddle/fluid/operators/activation_op.h]: same above

File: paddle/fluid/operators/activation_op.h [hunk 4]
```diff
@@ -1201,15 +1212,17 @@ inline void ExtractActivationDoubleGradTensor(
       *ddOut = ctx.Output<framework::Tensor>("DDOut");
     }
   }
-  PADDLE_ENFORCE(*ddX != nullptr,
-                 "Cannot get output tensor DDX, variable name = %s",
-                 ctx.OutputName("DDX"));
+  PADDLE_ENFORCE_NOT_NULL(
+      *ddX, platform::errors::NotFound(
+                "Cannot get output tensor DDX, variable name = %s",
```
Comment (chenwhql) [inline paddle/fluid/operators/activation_op.h]: same above, DDX is not tensor name?

File: paddle/fluid/operators/activation_op.h (hunk 2 above)
Comment (Steffy-zxf) [reply paddle/fluid/operators/activation_op.h]: done

File: paddle/fluid/operators/activation_op.h (hunk 3 above)
Comment (Steffy-zxf) [reply paddle/fluid/operators/activation_op.h]: done

File: paddle/fluid/operators/activation_op.h (hunk 4 above)
Comment (Steffy-zxf) [reply paddle/fluid/operators/activation_op.h]: done

**Review by chenwhql — APPROVED** [review]
LGTM

**Review by wawltor — APPROVED** [review]
LGMT

**Pull request merged**